packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -89,34 +62,32 @@ function getDefaultOnloadCallback(nodeKey, resolve, reject) {
   };
 }
 
-export function setOrCreateImage(
+/**
+ * Create image and fire callback after loaded.
+ *
+ * @param {Object} imageData Image data.
+ * @param {string} imageData.src Image source.
+ * @param {Object} imageData.height Image height.
+ * @param {Object} imageData.width Image width.
+ * @param {Function} getOnloadCallback Callback to fire after image is loaded.
+ * @return {Promise} Yields when the image is loaded.
+ */
+export function createImage(
   imageData,
   getOnloadCallback = getDefaultOnloadCallback
 ) {
-  const { src, id, height, width } = imageData;
+  const { src, height, width } = imageData;
   return new Promise((resolve, reject) => {
-    const NODE_KEY = getImgNodeKey(id);
-    let imgNode = document.body[NODE_KEY];
-    if (!imgNode) {
-      imgNode = document.createElement('div');
-      imgNode.id = getImgNodeId(id);
-      Object.assign(imgNode.style, STYLES);
-      document.body.appendChild(imgNode);
-      document.body[NODE_KEY] = imgNode;
-    }
-    imgNode.innerHTML = '';
-
     const img = new Image();
```

Comment:
Worth noting here, I would just try and use the `preloadImage` function here, but couldn't work out a way to do this cleanly.

Reply:
What was the issue?